libs/librepcb/libraryeditor/newelementwizard/newelementwizardpage_entermetadata.cpp

```diff
@@ -152,12 +152,19 @@ void NewElementWizardPage_EnterMetadata::btnChooseCategoryClicked() noexcept {
       return;
     }
   }
-  mContext.mElementCategoryUuid = categoryUuid;
+  mContext.mElementCategoriesUuid.clear();
+  mContext.mElementRootCategoryUuid = categoryUuid;
+  if (categoryUuid.has_value()) {
+    mContext.mElementCategoriesUuid.insert(*categoryUuid);
+  }
   updateCategoryTreeLabel();
 }
 
 void NewElementWizardPage_EnterMetadata::btnResetCategoryClicked() noexcept {
-  mContext.mElementCategoryUuid = tl::nullopt;
+  if (mContext.mElementCategoriesUuid.count()) {
+    mContext.mElementCategoriesUuid.clear();
+  }
```

Comment:
Can be simplified :)

```suggestion
  mContext.mElementCategoriesUuid.clear();
```